source/help/apps/desktop-guide.rst

```diff
@@ -85,9 +85,13 @@ Leave app running in notification center when application window is closed (Linu
 Use GPU hardware acceleration
     When enabled, the Mattermost user interface is rendered more efficiently but can lead to decreased stability for some systems. This setting is enabled by default. You must restart the app to change this setting.
     
-Dark Theme
-~~~~~~~~~~
+Dark Theme (Linux only)
```

Comment:
The way this heading reads is that dark mode is available on linux only, but that's not the case. Mac and Windows are just based on your OS.